app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
Code Style: Use `( )` while importing multiple classes/modules. It's a good practice to use parentheses for multiple imports from a single module. It will make your code cleaner and more readable. Here is how it should look: 

from app.errors import (
    VaccineError, 
    NotWearingMaskError
)
```